readthedocs/proxito/views/serve.py

```diff
@@ -265,40 +265,41 @@ def get(self, request, proxito_path, template_name='404.html'):
             version_slug=version_slug,
         )
 
-        storage_root_path = final_project.get_storage_path(
-            type_='html',
-            version_slug=version_slug,
-            include_file=False,
-            version_type=self.version_type,
-        )
-
-        # First, check for dirhtml with slash
-        for tryfile in ('index.html', 'README.html'):
-            storage_filename_path = build_media_storage.join(
-                storage_root_path,
-                f'{filename}/{tryfile}'.lstrip('/'),
+        if version_slug:
```

Comment:
the diff doesn't help, but this is just wrapping the storage calls under this big if